src/karmabot/settings.py

```diff
@@ -2,11 +2,12 @@
 import os
 import re
 from pathlib import Path
+from typing import Union
 
 from dotenv import load_dotenv
 
 
-def get_env_var(env_var: str, default: str | None = None) -> str:
+def get_env_var(env_var: str, default: Union[str, None] = None) -> str:
```

Comment:
weird although I was using Python 3.10 mypy (in pre-commit) said | was not ok so had to change to Union to even be able to commit

Reply:
Might have wanted `Optional`? Not sure and it's a mighty fine hair to split.  LGTM.

Reply:
Following Sebastian's opinion here ;)
https://fastapi.tiangolo.com/python-types/#using-union-or-optional